lib/jsonapi_plug/plug.ex

```diff
@@ -122,9 +122,9 @@ defmodule JSONAPIPlug.Plug do
 
   @doc false
   def config(%Conn{} = conn, _options) do
-    {options, assigns} = Map.pop!(conn.assigns, :jsonapi_plug)
+    options = conn.assigns.jsonapi_plug
 
-    %{conn | assigns: assigns}
+    conn
```

Comment:
This used to remove options from assigns. Is there any reason to not do it?

Reply:
No. I will revert this.